consensus/vbft/service.go

```diff
@@ -1043,6 +1064,30 @@ func (self *Server) onConsensusMsg(peerIdx uint32, msg ConsensusMsg, msgHash com
 			fromPeer: peerIdx,
 			msg:      msg,
 		}
+	case BlockSubmitMessage:
+		pMsg, ok := msg.(*blockSubmitMsg)
+		if !ok {
+			log.Error("invalid msg with submit msg type")
+			return
+		}
+		msgBlkNum := pMsg.GetBlockNum()
+		if msgBlkNum > self.GetCurrentBlockNo() {
```

Comment:
ignore the msg if node has  submitted the block?

Reply:
already added,in new commit